File: November 2022/26. Remove Duplicates from Sorted Array/solution.java
// Asked on 11-12-2022 (DD-MM-YYYY)
// Language: Java

class Solution {
    public int removeDuplicates(int[] nums) {
        Set<Integer> set = new HashSet<>();
        
        for(int i: nums){
            set.add(i);
        }
        
        return set.size();
    }
}

File: November 2022/26. Remove Duplicates from Sorted Array/solution.py
# Asked on 11-11-2022
# Language: Python

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if(len(nums)==0 or len(nums)==1):
            return len(nums)
        j = 0
        for i in range(0, len(nums)-1):
            if(nums[i] != nums[i+1]):
                nums[j] = nums[i]
                j = j+1
        nums[j] = nums[len(nums) - 1];
        j = j+1
        return j
